File: Lectures/CSC207/207_L0102/w01/primes.py
def isPrime(n):
	if n<2:
		return False
	else:
		for i in range(2,n):
			if n%i==0:
				return False
		return True

def primes(n):
	current = 2
	list_of_primes = []
	while len(list_of_primes)<n:
		if isPrime(current):
			list_of_primes.append(current)
		current = current + 1
	return list_of_primes

'''
for i in range(100):
	print(i,str(isPrime(i)))
'''

print(primes(10000)[9999])

File: Lectures/CSC207/207_L0102/w01/Primes.java
public class Primes {
	public static void main(String [] args){
		/**
		int i;
		for(i=0;i<100;i=i+1){
        		System.out.println(i+" "+isPrime(i));
		}
		**/
		System.out.println(primes(10000)[9999]);
	}
	public static boolean isPrime(int n){
        	if(n<2){
                	return false;
        	} else {
                	for(int i=2;i<n;i=i+1){
                        	if(n%i==0){
                                	return false;
				}
			}
                	return true;
		}
	}

	public static int [] primes(int n){
        	int current = 2;
		int num_primes=0;
        	int [] list_of_primes = new int[n];

        	while(num_primes<n){
                	if(isPrime(current)){
                        	list_of_primes[num_primes] = current;
				num_primes = num_primes+1;
			}
                	current = current + 1;
		}
        	return list_of_primes;
	}
}
